optaplanner-website-root/content/blog/2022/09/26/dynamic-scaling-of-optaplanner-workloads-on-openshift.adoc

```diff
@@ -0,0 +1,88 @@
+= Run OptaPlanner workloads on OpenShift, part II
+rsynek
+2022-09-26
+:page-interpolate: true
+:jbake-type: post
+:jbake-tags: cloud, openshift, kubernetes, scaling
```

Comment:
It's good to set a social media share image (see other blog posts for jbake attribute name), which shows up when the url is shared on twitter etc. It can be that demoArchitecture.png.

I'd recommend exporting the svg image as a png too (include both in the PR) and use the png in the document. Not all browsers deal with svg well in my experience, but that might have changed.